src/Condition.php

```diff
@@ -46,6 +47,10 @@ public function __construct($column, $value, $type = self::SAME, array $options
 		$this->value = $value;
 		$this->type = $type;
 		$this->options = $options;
+
+		if($type === self::CALLBACK && ( ! array_key_exists(self::CALLBACK, $options) || ! is_callable($options[self::CALLBACK]))){
```

Comment:
Please, can you remove the whitespaces like this: `(!array_key_exists(self::CALLBACK, $options) || !is_callable($options[self::CALLBACK]))`?